projects/commudle-admin/src/app/components/home/components/home-events/home-events-card/home-events-card.component.html

```diff
@@ -0,0 +1,27 @@
+<div *ngIf="community && event" class="home-events-card">
+  <div *ngIf="registrations > 0" class="home-events-attending">
+    <nb-icon icon="people"></nb-icon>
+    {{registrations}} Registered
+  </div>
+  <div [routerLink]="['/communities', community.slug, 'events', event.slug]" class="home-events-image clickable">
+    <img *ngIf="event.header_image_path" alt="{{event.name}}" src="{{event.header_image_path}}">
+    <div *ngIf="!event.header_image_path">
+      <app-skeleton-text [rows]="1"></app-skeleton-text>
+    </div>
+  </div>
+  <div class="home-events-date">
+    {{event.start_time | date:'MMM d'}}
```

Comment:
Should we use moment and display this date/time in the current user's local clock time?

Reply:
Sure, done!